app/pages/about/team-page.cjsx

```diff
@@ -122,6 +122,9 @@ to squeeze a bit of galaxy evolution science in.'''
         bio: '''Becky is an astrophysicist working towards her doctorate in Oxford. She is
         interested in how galaxies change over time and how we can track this evolution.
         Happy-go-lucky about outreach, the Zooniverse and everything.'''
+      robinSchaaf:
+        title: 'Mobile App Developer'
+        bio: '''Robin has a background in web development for educational institutions.  In her free team she enjoys board games and playing bass guitar.'''
```

Comment:
Sorry, bit too late, but there is a typo. Should be "In her free time" @robinschaaf